scripts/spack/packages/conduit/package.py

```diff
@@ -0,0 +1,552 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
What are we patching in this file?

Reply:
Not completely sure but it may just be that our spack SHA is from before 0.7.2 was added to the package

Reply:
There was no conduit 0.7.2 in our spack sha.  When I updated to the new spack develop I noticed failures due to the massive rewrite of the spack hdf5 package and ran away screaming.  I'll tackle that separately.